src/app/views/jobs/jobs.component.scss

```diff
@@ -48,8 +48,13 @@
 
 .job-list-container {
   background-color: $default-bg-color;
+  min-height: 1026px;
```

Comment:
Is this dependent on the container `max-width`? If so we should move this to a SASS-variable, if not leave it :)